docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -3,324 +3,778 @@ This guide is maintained in the main Quarkus repository
+
+
+When you use Hiberate ORM, you must create two classes that separate the state and its operations even though you would never do that for regular Java objects where state and methods are in the same class. In addition to requiring two classes, Hibernate ORM requires you to  inject the repository where you need to perform entity operations, which breaks your edit flow and requires you to get out of the code that you are  writing to set up an injection point before coming back to use it.
+
+You can use the Panache active record pattern with the `PanacheEntity` class to keep the model and the repository together.
+
+.Procedure
+
+. Add the Panache JPA extension dependency to your {ProductName} project `pom.xml` file:
++
+[source,java]
 ----
 <dependencies>
-    <!-- Hibernate ORM specific dependencies -->
     <dependency>
         <groupId>io.quarkus</groupId>
         <artifactId>quarkus-hibernate-orm-panache</artifactId>
     </dependency>
-
-    <!-- JDBC driver dependencies -->
-    <dependency>
-        <groupId>io.quarkus</groupId>
-        <artifactId>quarkus-jdbc-postgresql</artifactId>
-    </dependency>
 </dependencies>
 ----
 
-Then add the relevant configuration properties in `{config-file}`.
-
-[source,properties]
++
+. To define an entity, create a class that extends the `PanacheEntity` class, annotate it with `@Entity`, and add your
+columns as public fields. The following example creates and annotates the `Person` class:
++
+[source,java]
 ----
-# configure your datasource
-quarkus.datasource.url = jdbc:postgresql://localhost:5432/mydatabase
-quarkus.datasource.driver = org.postgresql.Driver
-quarkus.datasource.username = sarah
-quarkus.datasource.password = connor
-
-# drop and create the database at startup (use `update` to only update the schema)
-quarkus.hibernate-orm.database.generation = drop-and-create
+@Entity
+public class Person extends PanacheEntity {
+    public String name;
+    public LocalDate birth;
+    public Status status;
+}
 ----
-
-== Defining your entity
-
-To define a Panache entity, simply extend `PanacheEntity`, annotate it with `@Entity` and add your
-columns as public fields:
-
++
+. Optional: To specify that a field not persist, use the `@Transient` annotation. The following example annotates the `status` field with `@Transient`.
++
 [source,java]
 ----
 @Entity
 public class Person extends PanacheEntity {
     public String name;
     public LocalDate birth;
+
+    @Transient
     public Status status;
 }
 ----
-
-You can put all your JPA column annotations on the public fields. If you need a field to not be persisted, use the
-`@Transient` annotation on it. If you need to write accessors, you can:
-
+. Add any additional accessors. The following example adds an accessor that returns `name` as uppercase in the model and an accessor that stores all names in lowercase in the database.
+//COMMENT: Is there a different between 'in the model' and 'in the database'?
```

Comment:
Yes, the model refers to the Java representation, while the DB is where we store this externally.